helpers/util.sh

```diff
@@ -139,7 +139,7 @@ buildversion() {
 }
 
 yesnoall() {
-    if [ $BUILDALL = "y" ]; then
+    if [ -n "$BUILD_INDIVIDUAL_ASK" ] && [ ! $BUILD_INDIVIDUAL_ASK = "n" ]; then
```

Comment:
This currently fails shellcheck with SC2086.

It could be rewritten altogether as
```bash
if [ ! "$BUILD_INDIVIDUAL_ASK" = "n" ]; then
```

PS: is the following not preffered/the POSIX way? IMHO, it's easier to notice `!=` than `! ... =`
```bash
if [ "$BUILD_INDIVIDUAL_ASK" != "n" ]; then

Reply:
Changed as requested.